Client/Frontend/Browser/TabManager.swift

```diff
@@ -327,6 +327,13 @@ class TabManager : NSObject {
         return -1
     }
 
+    func getTab(webView: WKWebView) -> Browser? {
```

Comment:
I just realized that this functionality already exists with the `subscript` operator and removed this method.